packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -10090,7 +10090,7 @@ export default{
 			}
 		`);
 			expect(fs.readFileSync("dist/index.js", { encoding: "utf-8" })).toContain(
-				`import path from "path";`
+				`import path from "node:path";`
```

Comment:
This change is required as because unenv module resolution will always append the `node:` prefix